relay-log/src/setup.rs

```diff
@@ -84,6 +85,9 @@ pub struct SentryConfig {
     /// Enables reporting to Sentry.
     pub enabled: bool,
 
+    /// Sets the environment for this service.
+    pub environment: Option<Cow<'static, str>>,
```

Comment:
Why is this a `Cow`? Is there a use case where we generate `SentryConfig` with a static string?

Reply:
That's basically because `sentry-rust` has this type for environment in `ClientOptions` and I did not want to mess with it. 

Do you think we should change to something else? What would you propose?

Reply:
OK, that makes sense, all good then. I would have expected a `String` here and wondered where the `Cow` came from. I guess with `Cow` you could even deserialize without owning, although we don't do that for configs AFAIK.